### ISSUE TITLE
How can I reproduce WER in Librispeech?

### ISSUE THREAD

**Issue by FzuGsr** [hirofumi0810/neural_sp#177]
Hi,I want to reproduce Librispeech WER in this project.
I tried to run the script blstm_las with 3 gpus and  batch_size 128, but I couldn't get the result in readme.
My training results are as follows
```
2020-10-25 17:06:31,478 __main__ line:467 INFO: WER (dev_other_960_wpbpe10000, ep:1): 33.09 %
2020-10-25 18:45:39,793 __main__ line:467 INFO: WER (dev_other_960_wpbpe10000, ep:2): 24.93 %
2020-10-25 20:24:06,648 __main__ line:467 INFO: WER (dev_other_960_wpbpe10000, ep:3): 22.18 %
...
2020-10-27 12:21:00,003 __main__ line:467 INFO: WER (dev_other_960_wpbpe10000, ep:28): 13.30 %
2020-10-27 13:56:46,384 __main__ line:467 INFO: WER (dev_other_960_wpbpe10000, ep:29): 13.51 %
2020-10-27 15:30:51,522 __main__ line:467 INFO: WER (dev_other_960_wpbpe10000, ep:30): 13.34 %
```
Is there a problem? How to achieve the accuracy of readme (dev-other: 91.7).

![image](https://user-images.githubusercontent.com/17610399/97422777-4112f300-1949-11eb-8bf2-37abd665e5a6.png)
![image](https://user-images.githubusercontent.com/17610399/97422850-57b94a00-1949-11eb-85f2-f226b8d17f0c.png)
Look forward to your reply~
Thank you

**Comment by hirofumi0810** [comment]
@FzuGsr Did you use SpecAugment?

**Comment by FzuGsr** [comment]
> SpecAugment

no，I run run.sh directly

**Comment by hirofumi0810** [comment]
That is the reason. Please use `--conf2 data/spec_augment.yaml` and initialized parameters with the pre-trained model by setting `--asr_init path_to_your_model`.

**Comment by FzuGsr** [comment]
> --asr_init path_to_your_model

Thank you .
I will try it later.
